src/logging.ts

```diff
@@ -24,12 +24,13 @@ class Logger extends EventEmitter {
   }
 
   log(level: LogLevel, ...args: any) {
-    const now = new Date(Date.now()).toISOString();
+    const timestamp = Date.now();
+    const localDatetime = new Date(timestamp).toLocaleString();
```

Comment:
do we want to always log EST? Idk if deploying potentially in west coast regions would then make this default to PST.

Reply:
That is part of why I added that differentiation, it'll log out in the browser console in the user's timezone as a human friendly string, and get shipped to as the raw epoch timestamp.